app/controllers/export_controller.rb

```diff
@@ -119,6 +119,24 @@ def full_history_patients
           name: 'Vaccinations',
           tab: 'Vaccinations'
         },
+        close_contacts: {
```

Comment:
This is to do https://tracker.codev.mitre.org/browse/SARAALERT-1637 correct? I personally would prefer keeping that a separate PR if it isn't too difficult.

Reply:
this is actually something that the PO team and I specifically discussed, since we're updating the format now to include common_exposure_cohorts and there aren't too many code changes, I would prefer to just include it in here if that's ok